completions/_sops

```diff
@@ -0,0 +1,24 @@
+# 3rd party completion loader for commands using           -*- shell-script -*-
+# version 1 of the https://cli.urfave.org library.
```

Comment:
Not sure if there are substantial differences at the moment in the v1 vs v2 completions at the moment, but I initially looked into using the upstream main branch one as a template for `sops`  and didn't get it to work immediately, so thought we might want to add one for each version. That'll mean some more maintenance when tools upgrade their urfave/cli.

- v1: https://github.com/urfave/cli/blob/v1-maint/autocomplete/bash_autocomplete
- v2: https://github.com/urfave/cli/blob/v2-maint/autocomplete/bash_autocomplete
- to-be v3: https://github.com/urfave/cli/blob/main/autocomplete/bash_autocomplete